sway/commands/input/map_to_region.c

```diff
@@ -0,0 +1,56 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdlib.h>
+#include <string.h>
+#include <wlr/types/wlr_box.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+
+struct cmd_results *input_cmd_map_to_region(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "map_to_region", EXPECTED_EQUAL_TO, 4))) {
+		return error;
+	}
+	struct input_config *ic = config->handler_context.input_config;
+	if (!ic) {
+		return cmd_results_new(CMD_FAILURE, "No input device defined");
+	}
+
+	// This is used to clear the current output mapping.
+	ic->mapped_to_output = strdup("");
```

Comment:
This is weird. Can we set it to NULL and then update other code to not segfault? Do we have to free the old string?